greenboot-script-runner.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+run_required_scripts () {
+  required_greenboot_scripts=`ls /etc/greenboot.d/required/*.sh`
+  set +e
+  for script in $required_greenboot_scripts; do
+    bash $script
```

Comment:
An alternative to `set +e` for that is:

```
rc=0
$script || rc=$?
```

Up to you which one to use!

Reply:
I like it